README.md

```diff
@@ -174,6 +175,16 @@ fields
    currency_code: USD
 ```
 
+The `duration` field type requires a `time_unit` to be provided:
+With a value one of: milliseconds, seconds, minutes, hours
+
+```yaml
+fields
+ - name: Build time
```

Comment:
❓  I'm wondering if "build time" might be a difficult example to recognise unless you know that it's talking about CI/code compilation builds - is there another term we could use? e.g. `Time until support ticket resolved`?